src/tokens/dark-theme-color-semantic.js

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
I recommend all tokens moving to a single folder instead of spreading them across tokens and themes.